### ISSUE TITLE
Sample data

### ISSUE THREAD

**Issue by zh1peng** [YaleMRRC/CPM#3]
Hello,
Thank you for sharing your CPM code.
I did some tests with my own data, but the CPM did not seem to perform well as expected. I was wondering would it be possible to share some sample data (e.g., FC matrix and behavioral variable) on which CPM performed well?

**Comment by dadashkarimi** [comment]
Hi,
Its good to hear that you have tested our code. Would you be able to have access to HCP or PNC? we have tested on these standard collections and the results are promising. Can you share with us your spearman correlation?

**Comment by zh1peng** [comment]
Hi,
Sorry, I just got a chance to revisit this. I didn't mention I was using the python version of the CPM and this thread is specific to the python_CPM.

What I did before is performing the python_CPM on the FC matrix derived from task-based fMRI. The results were not good but I cannot find where they are. This time, I tested it with resting-state data and got the Rpos_mean: Nan and: Rneg_mean 0.14, which makes more sense now I think.

The following suggestions for the python CPM that author could consider, but these can be ignored as the code is acutally working.

1. a broadcast error is given when subject number cannot divided by the number of the fold in Kfold_CPM.  The size of the prediction of folds may different, which causes the problem when concatenating them. For example, 277 subjects and 10 fold did not work but 270 subjects and 10 fold worked for me. 
2.  It seems Line 147 should be `numsubs=X.shape[1]`. If I understand it correctly,  X or ipmat is in a 2D format of features (i.e.,node*node) * sub rather than 3D format as stated in the documentation.
3. results of the combination of the pos and neg models could be provided.
